modules/account-lib/src/coin/dot/keyPair.ts

```diff
@@ -12,18 +20,57 @@ export class KeyPair extends Ed25519KeyPair {
     super(source);
   }
 
+  /**
+   * Helper function to create the KeyringPair for signing a dot transaction.
+   *
+   * @returns {KeyringPair} dot KeyringPair
+   *
+   * @see https://polkadot.js.org/docs/api/start/keyring
+   */
+  protected createPolkadotPair(): KeyringPair {
+    const secretKey = this.keyPair.prv ? new Uint8Array(Buffer.from(this.keyPair.prv, 'hex')) : new Uint8Array();
+    const publicKey = new Uint8Array(Buffer.from(this.keyPair.pub, 'hex'));
+    return createPair({ toSS58: keyring.encodeAddress, type: TYPE }, { secretKey, publicKey });
+  }
+
+  /** @inheritdoc */
   getAddress(format?: AddressFormat): string {
-    throw new Error('Method not implemented.');
+    return this.createPolkadotPair().address;
   }
 
+  /**
+   * Getting the KeyringPair for signing a dot transaction.
+   *
+   * @returns {KeyringPair} dot KeyringPair
+   */
+  getSigningKeyPair(): KeyringPair {
+    return this.createPolkadotPair();
+  }
+
+  /** @inheritdoc */
   getKeys(): DefaultKeys {
```

Comment:
This method should return the keys in the protocol format:
https://github.com/BitGo/BitGoJS/blob/b6090cab84f867037d8f40267344426dfcc56244/modules/account-lib/src/coin/baseCoin/baseKeyPair.ts#L24